modules/k6/k6.go

```diff
@@ -0,0 +1,95 @@
+package k6
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+// K6Container represents the K6 container type used in the module
+type K6Container struct {
+	testcontainers.Container
+}
+
+// WithTestScript mounts the given script into the ./test directory in the container
+// and passes it to k6 as the test to run.
+// The path to the script must be an absolute path
+func WithTestScript(scriptPath string) testcontainers.CustomizeRequestOption {
+	return func(req *testcontainers.GenericContainerRequest) {
+		script := filepath.Base(scriptPath)
+		target := fmt.Sprintf("/tests/%s", script)
+		mount := testcontainers.ContainerMount{
+			Source: testcontainers.GenericBindMountSource{
+				HostPath: scriptPath,
+			},
+			Target: testcontainers.ContainerMountTarget(target),
+		}
+		req.Mounts = append(req.Mounts, mount)
+		req.Cmd = append(req.Cmd, target)
+	}
+}
+
+// WithCmdOptions pass the given options to the k6 run command
+func WithCmdOptions(options ...string) testcontainers.CustomizeRequestOption {
+	return func(req *testcontainers.GenericContainerRequest) {
+		req.Cmd = append(req.Cmd, options...)
+	}
+}
+
+// SetEnvVar sets an environment variable for the test script
```

Comment:
Let's be more expressive in the comment, as it's not modifying the env vars of the container request, as a casual reader could understand, but instead the real implementation. Something like 
```suggestion
// SetEnvVar sets an environment variable for the test script, that will end up as a command-line flag of the main program in the K6 container
```
You probably know better than me :)